packages/react-components/react-carousel-preview/src/components/CarouselNav/useCarouselNavStyles.styles.ts

```diff
@@ -23,6 +23,12 @@ const useStyles = makeStyles({
       borderRadius: tokens.borderRadiusMedium,
       ...shorthands.borderColor('transparent'),
     }),
+    borderRadius: tokens.borderRadiusXLarge,
+    marginInline: tokens.spacingHorizontalS,
+    marginBlock: 'auto',
+    paddingBlock: tokens.spacingHorizontalXS,
+    paddingInline: tokens.spacingHorizontalXS,
```

Comment:
`*Block` & `*Inline` are out of browser support matrix 😭

Reply:
> `*Block` & `*Inline` are out of browser support matrix 😭

Shot you a message about support here. We list 14.1 in our [matrix ](https://github.com/microsoft/fluentui/wiki/Browser-Support)but also have "latest 2 versions" which should mean our support is 16 and 17.